src/routes/blog/from-confusion-to-clarity.tiramisu

```diff
@@ -0,0 +1,105 @@
+title { From Confusion to Clarity: Timeleap Plugins }
+
+pullQuote {
+  "Simple can be harder than complex: You have to work hard to get your thinking clean to make it simple." – Steve Jobs
+}
+
+I remember the first time I came across the Timeleap Network — it immediately struck me as a cool and promising project. I thought, "This could be something big... I just wish it were easier to get started."
+
+Back then, the idea of some kind of wizard or streamlined setup felt like wishful thinking. But as time passed and I started working on a plugin myself, that early thought came full circle. We realized it wasn’t just a nice-to-have — it was a genuine need.
+
+That’s what led to this repo: a focused, developer-friendly space for writing and testing Timeleap plugins without needing to understand the entire system. It's built from real experience, and it's here to help others get moving faster
+
+title { 1. Why This Repo Exists, size = 2 }
+
+The goal of this monorepo is to make plugin development for Timeleap as approachable as possible. It removes the burden of understanding network internals by providing a Docker-based setup that handles the infrastructure for you.
+Included is a working inlineCode {Hello World} plugin, showcasing:
+
+list {
+  items = "How to register a plugin",
+  items = "How to expose a function hi",
+  items = "How to connect and interact with the network from a client"
+}
+
+image { src = "/images/blog/Timeleap-plugins.png", alt = "How the Timeleap plugin 'hello-world' works" }
```

Comment:
No capital letters in file names. Also, can you explain why the linter config is just a big white box? ALSO, why is there no description or explanation for this chart? We can't just drop a flowchart and assume people will get it.